Build 24.11.0rc with libmamba v2 compatibility

jaimergp opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

jaimergp committed:
Build 24.11.0rc

jaimergp commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

Commit:
MNT: Re-rendered with conda-build 24.9.0, conda-smithy 3.43.2, and conda-forge-pinning 2024.11.01.21.49.52

jaimergp commented on recipe/meta.yaml:
```diff
@@ -1,13 +1,13 @@
 source:
-  url: https://github.com/conda/conda-libmamba-solver/archive/{{ version }}.tar.gz
-  sha256: 77a78524719290468665c091cf073f2b97440bfea25c373105a997654063fdbe
+  url: https://github.com/conda/conda-libmamba-solver/archive/9dc525e62499e6ccbf10cd2053a523f3753dddb1.tar.gz
```
TODO before merge: Change to tag once available.

jaimergp replied:
```suggestion
  url: https://github.com/conda/conda-libmamba-solver/archive/{{ version }}.tar.gz
```

jaimergp commented on recipe/meta.yaml:
```diff
@@ -1,13 +1,13 @@
-  url: https://github.com/conda/conda-libmamba-solver/archive/{{ version }}.tar.gz
-  sha256: 77a78524719290468665c091cf073f2b97440bfea25c373105a997654063fdbe
+  url: https://github.com/conda/conda-libmamba-solver/archive/9dc525e62499e6ccbf10cd2053a523f3753dddb1.tar.gz
+  sha256: 626e568f7067c76064f686d28d29bcf380dca79adb56780013fd7b7b0c6d171b
```
```suggestion
  sha256: 1cc1cd12159ba67de40702c063662dcb6b8dcc643f6064977c8c004580f4ad23
```

jaimergp committed:
Use tag for archive

jaimergp committed:
Upload to label

MNT: Re-rendered with conda-build 24.9.0, conda-smithy 3.43.2, and conda-forge-pinning 2024.11.04.10.08.39

jaimergp committed:
Merge branch '24.11.0rc' of github.com:jaimergp/conda-libmamba-solver-conda-forge-feedstock into 24.11.0rc
